src/main/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupport.scala

```diff
@@ -154,6 +202,7 @@ trait DataUseRestrictionSupport extends LazyLogging {
       case (attr: AttributeName, AttributeBoolean(true)) => attr.name
     }.toSeq
 
+    //this isn't actually calling ontologyDAO?
```

Comment:
For what reason? It looks like everything needed here is in the workspace attribute values.